Builder/Admin/BaseBuilder.php

```diff
@@ -185,6 +189,12 @@ protected function createColumn($columnName, $withForms = false)
             if (array_key_exists('addFormOptions', $fieldOptions)) {
                 $column->setAddFormOptions($fieldOptions['addFormOptions']);
             }
+
+            if (array_key_exists('addFilterOptions', $fieldOptions)) {
+                $column->setAddFilterOptions($fieldOptions['addFilterOptions']);
+            } elseif (array_key_exists('addFormOptions', $fieldOptions)) {
```

Comment:
@sescandell then the whole purpose of the differentiation between filter options and form options is defied, making the addition useless. Especially in the case of different types.